packages/hardhat-core/test/internal/hardhat-network/provider/hardhat-network-options.ts

```diff
@@ -83,4 +85,58 @@ describe("Hardhat Network special options", function () {
       assert.isBelow(+secondBlock.timestamp, initialDate + 20);
     });
   });
+
+  describe("hardfork", function () {
+    describe("When not forking", function () {
+      useFixtureProject("hardhat-network-spurious-dragon");
+      useEnvironment();
+
+      it("should accept hard forks as late as chainnstart and use them", async function () {
```

Comment:
`chainstart`?

Reply:
That's the first hardfork according to `@ethereumjs/common`.

Reply:
ohh there was a typo